email_app/app/components/app_logs.js

```diff
@@ -17,23 +19,27 @@ export default class AppLogs extends Component {
     this.props.getLogPath()
       .then(path => {
         this.logPath = path.value;
-        this.readLogFile();
+        this.readLogFiles();
         this.setState({loading: false});
       })
       .catch(err => this.setState({loading: false, error: err.message}));
   }
 
-  readLogFile() {
+  readLogFiles() {
     if (!this.logPath) {
       this.setState({error: 'Log path not set'});
       return;
     }
     this.setState({loading: false});
     try {
-      const logData = fs.readFileSync(this.logPath);
-      this.setState({loading: false, logData: logData.toString()});
-    } catch(err) {
-      this.setState({loading: false, error: err.message});
+      const nativeLogData = fs.readFileSync(this.logPath);
+      const logMap = {};
+      Object.assign(logMap, { nativeLogData: nativeLogData.toString() });
+      const electronLogData = fs.readFileSync(`${process.cwd()}/${pkg.name}-nodejs.log`);
```

Comment:
Can we get the log filename from the app's state? i.e. have the app to push the log filename (when the log file is opened/created) into the Redux state, which can be then accessed from anywhere in the app including from here analogously to `this.props.getLogPath()` ?